### ISSUE TITLE
Pam's Harvestcraft is configured to nerf vanilla foods

### ISSUE THREAD

**Issue by urikkiru** [adam9899/Issues#306]
This becomes a little silly, as steak now gives the same food amount as say, berries. I'd recommend setting "B:enablevanillafoodBalance=false" in the general section of harvestcraft.cfg.

**Comment by ByteTemplar** [comment]
Why? It is set so, to push you making better foods with for example Pams harvestcraft and not just munch on basic steaks. This has added effect on eating different foods to get more hearts etc...

Or better yet, maybe make the Sandwich station (accepts baguette, just 2 wheat in crafting grid) that is one of the earliest quests and try putting few different things in it. Really easy way to get good food.

Or if it really bothers you so much, manually change it...

**Comment by burgstore** [comment]
as byte said, its an incentive to get you to make food that takes more than just killing an animal and cooking meat, baguettes and watermelon jelly sandwiches are good early game food. beyond that you should just start planting all the different types of crops.

**Comment by angryansari** [comment]
it Doesnt really do much for balance when Baguettes and most of the DungeonTactics stuff are just as easy to get as vanilla food

**Comment by urikkiru** [comment]
Agreed. Usually nerfing vanilla food isn't really sufficient. On paper it seems like a good idea, in practice it quickly just becomes an odd irritation to the player.

Instead, most packs have taken to just including https://www.curseforge.com/minecraft/mc-mods/spice-of-life-carrot-edition instead. This solves the issue of encouraging more diverse food choices, and thus validating the inclusion of Pam's Harvestcraft into your pack. Best of both worlds really.
